platform/linux-generic/include/odp_schedule_if.h

```diff
@@ -95,6 +95,7 @@ typedef struct {
 				   odp_schedule_group_info_t *);
 	void (*schedule_order_lock)(unsigned);
 	void (*schedule_order_unlock)(unsigned);
+	void (*schedule_order_unlock_lock)(uint32_t, uint32_t);
```

Comment:
Also lock / unlock function pointer prototypes need to be updated to uint32_t.

Reply:
Okay